src/Components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,8 +1,205 @@
-import React from 'react'
 
-export default function ChangePassword() {
+import { useTheme } from "@mui/material/styles";
+import Box from "@mui/material/Box";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import Photo from "../../assets/Group 3.png";
+import "../../App.scss";
+import TextField from "@mui/material/TextField";
+import { Link, useNavigate } from "react-router-dom";
+import { FormControl, Stack } from "@mui/material";
+import { useContext, useState } from "react";
+import { AuthContext } from "../Context/AuthContext/AuthContext";
+import { useForm } from "react-hook-form";
+import axios from "axios";
+import { toast } from "react-toastify";
+import CircularProgress from "@mui/material/CircularProgress";
+import { IChange } from "./ChangeInterfaces/ChangeInterfaces";
+import KeyboardBackspaceIcon from '@mui/icons-material/KeyboardBackspace';
+import Home from "@mui/icons-material/Home";
+
+
+export default function ChangePassword({ saveAdminData }) {
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<IChange>();
+  const [isLoading, setIsLoding] = useState(false);
+  const {baseUrl} = useContext(AuthContext);
+  const theme = useTheme();
+  // ****************** to RestPassword **********************
+  const onSubmit = (data: IChange) => {
+    setIsLoding(true);
+    axios
+      .post(`${baseUrl}/admin/users/change-password`, data)
```

Comment:
To change password, we need to be authorized. We need to add the request headers that contains the token.